composer.json

```diff
@@ -0,0 +1,33 @@
+{
+	"name": "pd/security-headers-nette-bridge",
+	"type": "library",
+	"description": "Napojení knihovny pro snadné nastavení bezpečnostních HTTP hlaviček do Nette",
+	"authors": [
+		{
+			"name": "Milan Pála",
+			"email": "milan.pala@peckadesign.cz"
+		}
+	],
+	"minimum-stability": "beta",
```

Comment:
Proč je tady potřeba beta stabilita? Je to jenom dočasně kvůli něčemu?

Reply:
Nedaří se mi natáhnout beta verzi knihovny při spouštění testů. Až bude release ostré verze, tak toto zmizí.